src/collections/src/NICollectionViewModel.m

```diff
@@ -181,6 +181,28 @@ - (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
   return nil;
 }
 
+#pragma mark - UICollectionViewDataSourcePrefetching
+
+- (void)collectionView:(UICollectionView *)collectionView prefetchItemsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths {
+  NSMutableArray* objects = [NSMutableArray array];
+  for (NSIndexPath* indexPath in indexPaths) {
+    id object = [self objectAtIndexPath:indexPaths];
+    [objects addObject:object];
+  }
+  
+  [self.delegate collectionViewModel:self collectionView:collectionView prefetchItemsAtIndexPaths:indexPaths withObjects:objects];
```

Comment:
To my knowledge, Nimbus's minimum OS version is currently iOS 9.0. If that is the case, then do we need availability guards around the invocations of the delegate methods that are marked available iOS 10.0?